src/allmydata/util/fileutil.py

```diff
@@ -110,9 +109,10 @@ def _crypt(self, offset, data):
         offset_big = offset // 16
         offset_small = offset % 16
         iv = binascii.unhexlify("%032x" % offset_big)
-        cipher = AES(self.key, iv=iv)
-        cipher.process("\x00"*offset_small)
-        return cipher.process(data)
+        cipher = aes.create_encryptor(self.key, iv)
+        # this is just to advance the counter
+        aes.encrypt_data(cipher, "\x00" * offset_small)
```

Comment:
`b"\x00"` right?

Reply:
Also this function looks similar to something else ... Oh yea.  `DecryptingConsumer.__init__`.  Maybe file a ticket for a future refactoring?